.github/workflows/ci.yaml

```diff
@@ -301,16 +311,19 @@ jobs:
       - name: Download build
         uses: actions/download-artifact@v3
         with:
-          name: cln-${ CFG }.tar.bz2
+          name: cln-gcc-dev1-exp1.tar.bz2
+
+      - name: Unpack build
+        run: tar -xvjf cln-gcc-dev1-exp1.tar.bz2
 
       - name: Test
         env:
           COMPAT: 1
           SLOW_MACHINE: 1
           TEST_DEBUG: 1
         run: |
-          tar -xjf cln-${{ matrix.CFG }}.tar.bz2
-          sed -i 's/VALGRIND=0/VALGRIND=1/g' config.vars 
+
+          sed -i 's/VALGRIND=0/VALGRIND=1/g' config.vars
           poetry run pytest tests/ -vvv -n 5 ${PYTEST_OPTS}  ${{ matrix.PYTEST_OPTS }}
```

Comment:
Frobbing config.vars here is a bad smell.... Um, can we do `VALGRIND=1 poetry...`?

Reply:
Initially we were using envvars in `pytest` exclusively but if I remember correctly at some point we switched to a strange combination of `config.vars` and envvars, so this was the simplest way of making absolutely sure we'd pick the right values for this config.

Happy to simplify if we can :-)